x/btcstaking/types/btc_staking_consumer_events.go

```diff
@@ -99,3 +99,13 @@ func CreateUnbondedBTCDelegationEvent(unbondedDel *BTCDelegation) (*BTCStakingCo
 
 	return event, nil
 }
+
+func CreateSlashedBTCDelegationEvent(slashedDel *BTCDelegation) *BTCStakingConsumerEvent {
+	return &BTCStakingConsumerEvent{
+		Event: &BTCStakingConsumerEvent_SlashedDel{
+			SlashedDel: &SlashedBTCDelegation{
+				StakingTxHash: slashedDel.MustGetStakingTxHash().String(),
```

Comment:
The extracted SK is still necessary for this packet? as the consumer side needs to verify the FP is indeed slashed

Reply:
> is it ok if we send slashing evidence instead of extracted SK and let the contract handle the extraction process (based on the full-validation feature flag), wdyt?

Yep sounds good to me, as this offsets computational cost on Babylon side to consumer side :+1: